extension-base/ctpGcfWrapper.js

```diff
@@ -0,0 +1,57 @@
+// Google Cloud Functions
+
+module.exports = {
+  ctpGcfWrapper: function(fn) {
+    return function(req, res) {
+      fn(req.body, ctpGcfResponse(res), console.log);
+    }
+  }
+}
+
+function ctpGcfResponse(res) {
+  return {
+    pass: passGcf(res),
```

Comment:
Nitpicky: but I would not postfix these with `*Gcf` as we already are in the `ctp-gfc-adapter` so they might just be a a `pass = createPass(res)`.